cassandra/datastax/cloud/__init__.py

```diff
@@ -117,6 +117,7 @@ def parse_cloud_config(path, cloud_config):
         cert_location = os.path.join(config_dir, 'cert')
         key_location = os.path.join(config_dir, 'key')
         config.ssl_context = _ssl_context_from_cert(ca_cert_location, cert_location, key_location)
+        config.pyopenssl_context = _pyopenssl_context_from_cert(ca_cert_location, cert_location, key_location)
```

Comment:
wondering if pyopenssl is a dependency of a lot of python packages (server, web etc.)... would be a nice to have to just create the ssl context we need.

Reply:
So are you saying rather than create both the standard ssl context and the pyopenssl context, you'd rather make a decision in the cloud config parsing code and just set a single context there? I considered that but thought this might be a little cleaner but also have no problem doing it that way. Would you rather I just set a single context?

Reply:
I don't care much if we have clearer and different ssl context attributes... I care more about constructing both when we can determine easily which one we need.